### PR TITLE
algos fetched from iiitv/algos

### LINKED ISSUE #23: fetch all algos from iiitv/algos
- fetch algos from ihttps://github.com/iiitv/algos/
- display algos in a temporary ui

For support take a look at https://github.com/mohitkyadav/algos-ngular
this is built with a very old angular js and it is not maintained anymore. But it still works.

### PR DESCRIPTION
**Pull request by prathmeshkhandelwal1**
Fixes #23 

Screenshot (103)

### CODE REVIEW

File: src/components/home/HomePage.js [hunk 1]
```diff
@@ -1,33 +1,75 @@
+/* eslint-disable prettier/prettier */
```
Comment (aashutoshrathi) [inline src/components/home/HomePage.js:1]: I think this line will not be approved by anyone. 👀

Comment (prathmeshkhandelwal1) [reply src/components/home/HomePage.js:1]: okk sir. I will remove it. xD

**Comment by prathmeshkhandelwal1** [comment]
@iiitv/project-maintainers can you review it plz?

File: src/components/home/HomePage.js [hunk 2]
```diff
@@ -2,32 +2,73 @@ import React, { Component } from 'react';
 import './HomePage.css';
 
 class HomePage extends Component {
```
Comment (mohitkyadav) [inline src/components/home/HomePage.js:4]: Can you make this a functional component?

File: src/components/home/HomePage.js [hunk 3]
```diff
@@ -2,32 +2,73 @@ import React, { Component } from 'react';
...
+  state = {
+    data: [],
+  };
+
+  componentDidMount() {
```
Comment (mohitkyadav) [inline src/components/home/HomePage.js:9]: and use `useEffect` to replace `componentDidMount` in functional component

**Review by mohitkyadav — CHANGES_REQUESTED** [review]

**Comment by prathmeshkhandelwal1** [comment]
Sir i tried many times to convert it into functional component but it is giving many error and also app is crashing many times.
Screenshot (104)

i created a functional component still it is giving error like you should use hooks in functional component.
Is it okk for this to be class component? (and also i m not good at react-hooks and i never used it in my projetcs)
@mohitkyadav
@iiitv/project-maintainers @AmanRaj1608

**Comment by mohitkyadav** [comment]
@AmanRaj1608 can you help @prathmeshkhandelwal1 ?

**Comment by AmanRaj1608** [comment]
@mohitkyadav Sure Sir.

**Comment by mohitkyadav** [comment]
@prathmeshkhandelwal1 resolve the conflicts and you will be able to create functional components now.

**Pull request closed without merging**